lib/ex.coffee

```diff
@@ -106,16 +107,30 @@ class Ex
   tabp: => @tabprevious()
 
   edit: (range, filePath) ->
-    filePath = fs.normalize(filePath.trim())
+    filePath = filePath.trim()
+    if filePath[0] is '!'
+      force = true
+      filePath = filePath[1..].trim()
+    else
+      force = false
```

Comment:
This, also probably worth moving the check for multiple filenames to `getFullPath` and making it a bit more sophisticated (allow for `\ ` in filenames, but not for `\\ `).

But I basically wanted to limit this PR's scope to adding specs. I mainly modified the commands because writing these specs was a bit of a pain already and I couldn't be bothered to rewrite them :flushed: